src/main/java/de/teamlapen/vampirism/entity/EntityVampirism.java

```diff
@@ -168,6 +178,9 @@ public void readEntityFromNBT(NBTTagCompound nbt) {
                 this.setMoveTowardsRestriction(nbt.getInteger("moveHomePrio"), true);
             }
         }
+        if (nbt.hasKey("activeAction")) {
+            entityActionHandler.resetAction(EntityActionManager.actionsByID.get(nbt.getInteger("activeAction")));
```

Comment:
the map get could return null (if an action should be removed in a future version). Probably fine since resetAction accepts null values.